lib/src/callback_dispatcher.dart

```diff
@@ -9,6 +9,7 @@ import 'package:flutter/services.dart';
 import 'package:geofencing/src/geofencing.dart';
 import 'package:geofencing/src/location.dart';
 
+@pragma('vm:entry-point')
```

Comment:
needs to be added so that native code can call back this dispatcher to forward the geofnce event triggers to the app